### ISSUE TITLE
Broken Scss highlighting

### ISSUE THREAD

**Issue by XhmikosR** [alecthomas/chroma#273]
We are using Hugo 0.57.2 and we hit the following issue:

```
{{< highlight scss >}}
.custom-element {
  color: color-yiq(#000); // returns `color: #fff`
}
{{< /highlight >}}
```

This is transformed wrongfully to

```html
<div class="highlight"><pre class="chroma"><code class="language-scss" data-lang="scss"><span class="nc">.custom-element</span> <span class="p">{</span>
  <span class="nt">color</span><span class="nd">:</span> <span class="nt">color-yiq</span><span class="o">(</span><span class="nn">#000</span><span class="o">);</span> <span class="o">//</span> <span class="nt">returns</span> <span class="err">`</span><span class="nt">color</span><span class="nd">:</span> <span class="nn">#fff</span><span class="err">`</span>
<span class="p">}</span></code></pre></div>
```

I think Hugo is using chroma 0.6.4, I don't suppose this was fixed in recent versions?

If I remove the backticks from the comment, the `err` class goes away but then the comment isn't detected as such.

**Comment by alecthomas** [comment]
Fixed in 2b493e25561bf41593104280a2ef68e1e357770f

**Comment by XhmikosR** [comment]
Thanks for the quick fix!
